skins/cat17/faq-page/src/Faq.vue

```diff
@@ -12,14 +12,16 @@
  	<h2>{{ messages.page_title }}</h2>
  	<h5>{{ messages.page_subtitle }}</h5>
  	<div class="row">
- 	<div class="col-xs-12 col-sm-9">
+ 	<div class="col-xs-12 col-sm-8">
 	  	<search-bar :messageSearch="messages.search"></search-bar>
 		<div class="form-shadow-wrap">
-			<h2 v-if="!isPreview" class="title space-below underlined">{{ topicTitle }}</h2>
+			<h2 v-if="!isPreview" class="title">{{ topicTitle }}</h2>
 			<ul>
-				<li v-for="content in page" v-bind:class="[ isPreview ? 'preview' : 'topic', 'underlined' ]">
-					<question :content="content.question"></question>
-					<answer :hiddenContent="content.hidden_text" :visibleContent="content.visible_text" :messages="messages" :isPreview="isPreview"></answer>
+				<li v-for="(content, index) in page" v-bind:class="[ isPreview ? 'preview' : 'topic' ]">
+					<question v-on:question-opened="setOpenQuestionId( $event )"
+							  :content="content"
+							  :idx="index"
+							  :visible="openQuestionId"></question>
```

Comment:
This property name is a little bit confusing to me because it can be read as a boolean. Maybe call it 
`visibleQuestion` or `visibleQuestionId`?